### PR TITLE
solves hover effect issue in dark mode

### LINKED ISSUE #10351: [Docs] Dark mode bug
### Current State
Dark mode bug:
image

### Desired State

---

### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by ayush844**
**Notes for Reviewers**

This PR fixes #10351 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit fdb5583676fafad8504af3ca1cc1ba957d9826cd at: https://65da4e9c3712083b3a006748--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 8ea44ed35c84f9de37eb67af056f4ec5c6a64bc7 at: https://65da50074e315f3c2aaf14c6--meshery-docs-preview.netlify.app

**Comment by vishalvivekm** [comment]
@ayush844 

Let's discuss it on the websites call.
Please add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima), if you would :)

File: docs/_layouts/past-results-layout.html [hunk 1]
```diff
@@ -54,9 +54,8 @@ <h2>{{page.subheading}} Integration Tests</h2>
 </script>
 
 <style>
-  td:hover,
-  tr:hover {
-    background-color: #ccfff9;
+  td.details table tr:hover td{
+    background-color: var(--hidden-details-data-color) !important;
```
Comment (tofikwest) [inline docs/_layouts/past-results-layout.html:58]: why did you use "!important"?

Comment (ayush844) [reply docs/_layouts/past-results-layout.html:58]: > why did you use "!important"?

@tofikwest 
!important is ensuring that the specified styles (background-color and cursor) are applied, even if there are other conflicting styles defined elsewhere.
this was present almost everywhere where we are using variable colors in table.scss, so i used it here.

**Comment by l5io** [comment]
🚀 Preview for commit 9b2d06d1f57ba17b92df0c81323d4b17090703c8 at: https://65dc8f350acc2cbfda1f5352--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
@tofikwest, thank you for reviewing. This is most helpful. 👍

**Comment by leecalcote** [comment]
@ayush844, it's your first PR on the project! Exciting. 🎉

**Comment by l5io** [comment]
🚀 Preview for commit ea9c61efd4a8a8fdc2c5952d61107c630ff224f6 at: https://65dd4b1a9b6a8869bc4a8704--meshery-docs-preview.netlify.app

**Review by iArchitSharma — APPROVED** [review]
LGTM!
thanks @ayush844

**Pull request merged**